src/core/js/accessibility.js

```diff
@@ -273,8 +273,12 @@ define([
                  //Remove button
                 this.$accessibilityToggle.remove();
             }
-
-            if (!Modernizr.touch || this.isActive() || Adapt.config.get("_accessibility")._isDisabledOnTouchDevices) return;
+            
+            var config = Adapt.config.get("_accessibility");
+            // Backwards compatibility for _isDisabledOnTouchDevices
+            var isEnabledOnTouchDevices = config._isEnabledOnTouchDevices || (config._isDisabledOnTouchDevices === false);
```

Comment:
Old courses which don't support _isEnabledOnTouchDevices but which do have _isDisabledOnTouchDevices set to false will still work as expected.